components/social/org.wso2.carbon.social.sql/src/main/java/org/wso2/carbon/social/sql/SocialUtil.java

```diff
@@ -44,6 +44,12 @@ public static String getTenantDomain() {
 
 	}
 
+    public static String getTenantedUsername() {
```

Comment:
The formatting issue appears to be with the existing code.I have verified by checking with the wso2 IDE style